doc/dune

```diff
@@ -10,8 +10,10 @@
   ;   + tools/coqdoc/coqdoc.css
   (package coq)
   (source_tree sphinx)
-  (source_tree tools))
- (action (run sphinx-build -j4 -b html -d sphinx_build/doctrees sphinx sphinx_build/html)))
+  (source_tree tools)
+  (env_var SPHINXWARNOPT))
```

Comment:
This dependency should be automatically added by Dune as `%{env:...}` does appear in the run rule.

Reply:
Thanks, I didn't know that.